paper-server/src/main/java/org/bukkit/craftbukkit/block/CraftVault.java

```diff
@@ -24,4 +39,114 @@ public CraftVault copy() {
     public CraftVault copy(Location location) {
         return new CraftVault(this, location);
     }
+
+    @Override
+    public double getActivationRange() {
+        return this.getSnapshot().getConfig().activationRange();
+    }
+
+    @Override
+    public void setActivationRange(final double activationRange) {
+        Preconditions.checkArgument(Double.isFinite(activationRange), "deactivation range must not be NaN or infinite");
+        Preconditions.checkArgument(activationRange <= getDeactivationRange(), "New activation range (%f) must be less or equal to deactivation range (%f)".formatted(activationRange, getDeactivationRange()));
```

Comment:
```suggestion
        Preconditions.checkArgument(activationRange <= this.getDeactivationRange(), "New activation range (%s) must be less or equal to deactivation range (%s)", activationRange, this.getDeactivationRange());
```
Not sure if change but Preconditions support format then not sure why need use the String format.

Reply:
Didn't think to check if it had that, will update it for both